Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -676,6 +684,15 @@ const ScrollView = createReactClass({
     this.scrollResponderHandleScroll(e);
   },
 
+  _handleLayout: function(e: Object) {
+    if (this.props.invertStickyHeaders) {
```

Comment:
Fast path here since we only need the height of the ScrollView for the inverted case which is pretty rare.